scripts/test-upgrade.sh

```diff
@@ -29,3 +30,15 @@ RPCD_DIR="$BASE_DIR/rpcd"
 #        to fail to run.
 cd ${RPCD_DIR}/playbooks
 ansible -m file -a "path=/var/www/repo/openstackgit/horizon-extensions state=absent" repo_all
+
+# TASK #2
+# Bug: https://github.com/rcbops/u-suk-dev/issues/215
+# issue: Enable the Horizon panels for LBaaSv2 if the environment has LBaaSv2
+#        enabled already. Check for anything matching "LoadBalancerPluginv2"
+#        in the OSA config and enable the Horizon panels via the RPC defaults
+#        config.
+if grep -q "LoadBalancerPluginv2" /etc/openstack_deploy/user_variables.yml ; then
+    if ! grep -q "^horizon_enable_neutron_lbaas" "${OSA_VARS}" ; then
+        echo "horizon_enable_neutron_lbaas: True" | tee -a "${OSA_VARS}"
```

Comment:
I would have thought it was a bit late in the day to be changing behaviour in stable/mitaka however if upstream with accept the change this sounds like the ideal solution to me. 

Alternatively, I think this is a documentation task, we just need to document that when setting the neutron var to enable LBaaSv2 it is recommended to also set "horizon_enable_neutron_lbaas: True". These vars should be set in /etc/openstack_deploy/user_osa_variables_overrides.yml.

As to testing, I don't believe we currently test this in the AIO, if it's still in tech preview and so not the default, I would be inclined to leave it out and rely on QE. We could also look to add additional AIO jobs. What is the likelihood that enabling it could hide issues that present when it's not enabled?